Packs/RegexReplace/Scripts/RegexReplace/RegexReplace.yml

```
@@ -0,0 +1,51 @@
+args:
+- description: Text to match against
+  name: value
+  required: true
+- description: Regex pattern to search
+  name: regex
+  required: true
+- description: Template string to format patterns matched with regex
+  name: output_format
+  required: true
+- auto: PREDEFINED
+  default: true
```

```suggestion
  default: false
```
Why do you want this argument to be default? 
that means that if you are calling the script without specifying the args (!RegexReplace something), the something value will automatic populate this argument